share_app/build.gradle

```diff
@@ -56,4 +56,7 @@ dependencies {
 
     // evernote's job scheduler
     implementation "com.evernote:android-job:1.2.5"
+
+    //MPAndroidChart
+    implementation 'com.github.PhilJay:MPAndroidChart:v3.0.3'
```

Comment:
Please add library information in open_source_licenses.html under assets

Reply:
Also, please replace the statistics screenshots from the readme file, so that it also gets updated with the new change.